x/sequencer/keeper/slashing.go

```diff
@@ -38,6 +38,14 @@ func (k Keeper) Slashing(ctx sdk.Context, seqAddr string) error {
 	// in case we are slashing an unbonding sequencer, we need to remove it from the unbonding queue
 	if oldStatus == types.Unbonding {
 		k.removeUnbondingSequencer(ctx, seq)
+	} else {
```

Comment:
as there would be various places in the app where sequencer can get jailed (and hence have his unbonding queue cleared), thinking if it makes more sense to simply create this hook, hook into it and let callers invoke it vs updating it after.